packages/medusa/src/api/routes/admin/order-edits/delete-order-edit.ts

```diff
@@ -0,0 +1,71 @@
+import { EntityManager } from "typeorm"
+import { OrderEditService } from "../../../../services/order-edit"
+
+/**
+ * @oas [delete] /order-edits/{edit_id}
```

Comment:
suggestion: should we use `id` instead of `edit_it` since we are under the order-edits already, such what we have done for the batch-job for example